src/olympia/devhub/tasks.py

```diff
@@ -110,9 +110,9 @@ def create_version_for_upload(addon, upload,
         # review since listed and sideload aren't supported for creation yet.
         if addon.status == amo.STATUS_NULL:
             if disallow_preliminary_review:
-                addon.update(status=amo.STATUS_PUBLIC)
+                addon.update(status=amo.STATUS_NOMINATED)
```

Comment:
Woah, ok. I followed the thread to make sure the same method was used, but I missed that `submit` parameter. Sigh.